Added images for analysis

BMatrix opened the pull request:
These images are used in the wiki page for the analysis

BMatrix committed:
Added images for analysis

These images are used in the wiki page for the analysis

F0xedb reviewed (changes requested):
All on images should be removed from the image directory and moved to a more appropriate directory

BMatrix committed:
Created folder design

I moved the .drawio and .mup files that were used to create some diagrams to a new folder called designs.

BMatrix commented:
I moved the .drawio and .mup files that were used to create some diagrams to a new folder called designs.

F0xedb reviewed (approved):
The comments on the last review have been fixed. This pull request is now allowed to merge
